stairway/src/main/java/bio/terra/stairway/queue/QueueMessage.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.stairway.queue;
+
+import bio.terra.stairway.impl.StairwayImpl;
+
+abstract class QueueMessage {
+  static final String FORMAT_VERSION = "1";
+
+  abstract boolean process(StairwayImpl stairway) throws InterruptedException;
```

Comment:
add comments for the class (and when it should be subclassed) as well as for `process`